NohanaImagePicker/AlbumListViewController.swift

```diff
@@ -178,37 +185,12 @@ class AlbumListViewController: UITableViewController, EmptyIndicatable, Activity
         }
     }
 
-    // MARK: - Storyboard
-
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        guard let sectionType = AlbumListViewControllerSectionType(rawValue: tableView.indexPathForSelectedRow!.section) else {
-            fatalError("Invalid section")
-        }
-        switch sectionType {
-        case .moment:
-            let momentViewController = segue.destination as! MomentViewController
-            momentViewController.nohanaImagePickerController = nohanaImagePickerController
-        case .albums:
-            switch segue.identifier {
-            case "toAssetListViewController":
-                let assetListViewController = segue.destination as! AssetListViewController
-                assetListViewController.photoKitAssetList = photoKitAlbumList[tableView.indexPathForSelectedRow!.row]
-                assetListViewController.nohanaImagePickerController = nohanaImagePickerController
-            case "toAssetListViewSelectableDateSectionController":
-                let assetListSelectableDateSectionController = segue.destination as! AssetListSelectableDateSectionController
-                assetListSelectableDateSectionController.photoKitAssetList = photoKitAlbumList[tableView.indexPathForSelectedRow!.row]
-                assetListSelectableDateSectionController.nohanaImagePickerController = nohanaImagePickerController
-            default:
-                fatalError("unexpected segue identifer")
-            }
-        }
-    }
-
     // MARK: - IBAction
 
-    @IBAction func didPushCancel(_ sender: AnyObject) {
-        if let nohanaImagePickerController = nohanaImagePickerController {
-            nohanaImagePickerController.delegate?.nohanaImagePickerDidCancel(nohanaImagePickerController)
```

Comment:
Is there an alternative delegate  function?

Reply:
Modify it to be called by RootViewController.